apps/api/src/app/inbox/usecases/unsnooze-notification/unsnooze-notification.usecase.ts

```diff
@@ -126,7 +126,5 @@ export class UnsnoozeNotification {
     if (!isSnoozeEnabled) {
       throw new NotImplementedException();
     }
-
-    // TODO: add per environment feature on/off on integration settings
```

Comment:
I think we don't need to check if snooze feature is enabled for unsnooze, if you have any snoozed notifications you should be able to unsnooze them using API even on free tier.

Reply:
Correct.